ci_scripts/shared-build-context/build.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+set -xe 
+
+if [ -z ${BUILDER_NAME+x} ]; then
+  echo "Error: Environment variable BUILDER_NAME is unset. It should be set to
+  the first child folder name in \`/builder\` and is normally set in the Dockerfile
+  for this builder. Examples for values are \`ubuntu-16_04\` or \`ubunut-18_10-rpgp\`.";
+  exit 0
+fi
+
+
+# BUILD DESKTOP
+cd /build
+rm -rf node_modules
+# install dependencies and compile deltachat-node with some specific flags
+PKG_CONFIG_PATH=/opt/DeltaChat/rpgp/lib/pkgconfig/ npm install
+
+# recompile deltachat-core with prefix
+cd /build/node_modules/deltachat-node/deltachat-core/
+rm -rf builddir
+PKG_CONFIG_PATH=/opt/DeltaChat/rpgp/lib/pkgconfig/ meson -Drpgp=true --prefix /opt/DeltaChat/libdeltachat builddir
+
+# globally install libdeltachat
+cd builddir
+ninja install
+
+# recompile node bindings
+cd ..
+npm run rebuild-bindings
+
+
+# actually build desktop files
+cd /build
+#npm run build
+
+
+# build .deb with electron builder (rebuilds deltachat-core for electron)
+PKG_CONFIG_PATH=/opt/DeltaChat/rpgp/lib/pkgconfig/ npx electron-builder -c /shared-build-context/electron-builder-ubuntu.json
```

Comment:
you could use ``export PKG_CONFIG_PATH=...`` so that the below recurrence doesn't need to specify the same path again.